data/shaders/opengl/geosphere_star.frag

```diff
@@ -1,8 +1,7 @@
-// Copyright � 2008-2020 Pioneer Developers. See AUTHORS.txt for details
+// Copyright � 2008-2020 Pioneer Developers. See AUTHORS.txt for details
```

Comment:
what going on with the © symbol?

Reply:
I honestly have no idea. I didn't touch that line at all. The only thing I can think of is that the file was originally written in utf-16 or something?